scripts/release/release.sh

```diff
@@ -32,6 +32,7 @@ version() {
     node_modules/.bin/lerna version --conventional-commits --no-commit-hooks --no-git-tag-version --no-push --allow-branch="${GIT_BRANCH}" --yes
   else
     printf "%s\n" "Releasing"
+    node_modules/.bin/lerna version prerelease --preid alpha --force-publish='@ovh-ux/shell;@ovh-ux/url-builder;@ovh-ux/manager-react-components;@ovh-ux/manager-react-shell-client'
```

Comment:
We need to conventional-prerelease to auto version based on the conventional-commits.